src/tmpfiles/tmpfiles.c

```diff
@@ -219,7 +219,7 @@ typedef struct Context {
 } Context;
 
 STATIC_DESTRUCTOR_REGISTER(arg_include_prefixes, freep);
```

Comment:
can you switch over arg_include_prefixes too? it would suck if the two very similar fields had different memory management